tests/dragonfly/cluster_test.py

```diff
@@ -2580,3 +2580,68 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     assert (await StaticSeeder.capture(nodes[1].client)) == start_capture
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "cache_mode": "true"})
```

Comment:
do you want to seprate the test from the fix PR?
I approve the fixes for dfly code but the test needs a little more work